sources/packages/api/src/route-controllers/education-program-offering/education-program-offering.controller.ts

```diff
@@ -221,13 +224,15 @@ export class EducationProgramOfferingController {
     @Param("programId") programId: number,
     @Param("programYearId") programYearId: number,
     @Query("selectedIntensity") selectedIntensity: OfferingIntensity,
```

Comment:
can we please move the selectedIntensity query parameter down and make includeInActivePY as mandatory query parameter, so that pass either true or false for the includeInActivePY. SO that we may not use undefined when calling this method .

Reply:
since `includeInActivePY` is a query parameter, there is no need to send, if it's `undefined` @guru-aot. the query parameters are optional